api/api/models/__init__.py

```diff
@@ -75,7 +77,7 @@ def send_confirm_email(self) -> int:
 
     def send_reset_password_email(self) -> int:
         """sends password-reset email"""
-        url = f'https://voron.sh/reset?token={self.tmp_token}'
+        url = f'https://voron.djnn.sh/reset?token={self.tmp_token}'
```

Comment:
why not using environment variable ?

Reply:
this is not related to this issue, but i agree

Reply:
This should be addressed in #12 in my opinion